utils/R/read.output.R

```diff
@@ -123,8 +123,6 @@ read.output <- function(runid, outdir, start.year=NA,
 
   if(!nofiles){
     for(ncfile in ncfiles) {
-      valid.nc <- tryl(nc_open(ncfile))
```

Comment:
You can put the whole block of code, up to and including the nc_close inside that try.